Lib/test/test_codecmaps_cn.py

```diff
@@ -4,22 +4,23 @@
 #
 
 from test import multibytecodec_support
+from test import support
 import unittest
 
 class TestGB2312Map(multibytecodec_support.TestBase_Mapping,
                    unittest.TestCase):
     encoding = 'gb2312'
-    mapfileurl = 'http://www.pythontest.net/unicode/EUC-CN.TXT'
+    mapfileurl = f'{support.TEST_HTTP_URL}/unicode/EUC-CN.TXT'
```

Comment:
Please revert this change. I would like to be able to modify TEST_HTTP_URL without breaking this URL.

Reply:
just this one? or the complete commit?